jokes-cli.js

```diff
@@ -0,0 +1,147 @@
+#! /usr/bin/env node
+const jokes      = require('./jokes.js')
```

Comment:
What would be intresting is using the already present module as dependency, the stand alone jokes.dz won't be useful then, that's why i used it as dependency over my package.json file. so please change that :)